compiler-plugin-test/src/test/resources/project_3/Ballerina.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+org = "root"
+name = "project_3"
+version = "0.1.0"
+
+[persist]
+datastore = "google-sheets"
```

Comment:
```suggestion
datastore = "googlesheets"
```

Reply:
We decided to remove `-` from the datasource names. please do the change

Reply:
Fixed in [071d253](https://github.com/ballerina-platform/module-ballerina-persist/pull/196/commits/071d253b42c588d7d79babd70d53b07a2265cb0f)